core/unix/os.c

```diff
@@ -1087,6 +1091,28 @@ get_application_name(void)
     return get_application_name_helper(false, true /* full path */);
 }
 
+/* Sets pointers to the application's command-line arguments. These pointers are then used
+ * by get_application_cl_args().
```

Comment:
`get_application_cl_args` does not exist

Reply:
Done.